### ISSUE TITLE
Complete fork?

### ISSUE THREAD

**Issue by alexgorbatchev** [vdepizzol/jquery-tokeninput#6]
I'm thinking about making this a full fork. It feels like your features are far ahead of the official lib and they don't seem to be maintaining it. I would really like to convert all names to proper camel casing, it hurts me to see php naming in a js lib :)

what do you think?

**Comment by vdepizzol** [comment]
Hey Alex! I think you are more than welcome to do such proposed changes! I think we can already push changes that won't be compatible with original jquery-tokeninput :).

Thank you!

PS: camel case sounds much better in JS, indeed!

**Comment by alexgorbatchev** [comment]
If we do convert to camel, we basically won't be able to merge anything that's forked from the original because diff would be too great :( it would be very painful.

I'm currently trying to figure out what would be nice to merge, before potentially doing this.

**Comment by robzolkos** [comment]
+1  great idea

**Comment by vdepizzol** [comment]
Yes, that's true. Specially now that there is some begin of work to make add/remove/clear methods: 7ab7e79f17a4f461d9ce

**Comment by loopj** [comment]
Hey guys, I've implemented a ton of changes similar to those in this fork in the original version, I'm more than happy to accept pull requests for individual features and bug fixes in the original. Any chance we could sync these versions back up again?

**Comment by vdepizzol** [comment]
Hey, @loopj. I'll be more than happy to sync the versions and request some pulls. I'm travelling right now and will look into this when get back.

**Comment by loopj** [comment]
Awesome thanks @vdepizzol let me know if you need any info/assistance.

**Comment by ches** [comment]
+1 for consolidation efforts :-)
